pkg/controller/importconfig/cluster_info.go

```diff
@@ -317,3 +323,12 @@ func getBootstrapSAName(clusterName string) string {
 	}
 	return bootstrapSAName
 }
+
+func getBootstrapSASecretName(clusterName string) string {
+	bootstrapSASecretName := fmt.Sprintf("%s-%s", clusterName, bootstrapSASecretSuffix)
+	if len(bootstrapSASecretName) > 63 {
+		return fmt.Sprintf("%s-%s", clusterName[:63-len("-"+bootstrapSASecretSuffix)], bootstrapSASecretSuffix)
```

Comment:
this needs doc. Why secret name is this?

Reply:
remove this, I use the service account name as the secret name